identity/ideth/factory.go

```diff
@@ -107,12 +109,12 @@ func (s *factory) CreateIdentity(ctx context.Context) (did *id.DID, err error) {
 		return nil, err
 	}
 
-	identityAddress, err := s.CalculateIdentityAddress(ctx)
+	calcIdentityAddress, err := s.CalculateIdentityAddress(ctx)
```

Comment:
If we get the `identityAddress` why calculating the address at all?

We could just use  createdAddr?

Reply:
We need to pass some identity to the tx manager. And it gives us good visibility when a collision happened.